service/src/main/frontend/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= productName %></title>
```

Comment:
It might be nice to add the dictionary service version somewhere on the page - not necessarily in the title.